registry/recipes/nginx.md

```diff
@@ -166,23 +166,24 @@ Review the [requirements](/registry/recipes/index.md#requirements), then follow
 5.  Create the compose file. Paste the following YAML into a new file called `docker-compose.yml`.
 
     ```yaml
-    nginx:
-      # Note : Only nginx:alpine supports bcrypt.
-      # If you don't need to use bcrypt, you can use a different tag.
-      # Ref. https://github.com/nginxinc/docker-nginx/issues/29
-      image: "nginx:alpine"
-      ports:
-        - 5043:443
-      links:
-        - registry:registry
-      volumes:
-        - ./auth:/etc/nginx/conf.d
-        - ./auth/nginx.conf:/etc/nginx/nginx.conf:ro
-
-    registry:
-      image: registry:2
-      volumes:
-        - ./data:/var/lib/registry
+    version: "3"
```

Comment:
@glours How do we represent the Compose spec here instead of the legacy version 3?

Reply:
Is there any update here?

Reply:
I believe we still need to remove these docs from the repository (as the docs were transferred to the upstream project); if someone can open a pull request for the upstream instead?

Upstream file is in this repo and location on GitHub; https://github.com/distribution/distribution/blob/main/docs/recipes/nginx.md

Reply:
Thanks for the hint! I've opened https://github.com/distribution/distribution/pull/3872. I'll close this one then.